infrastructure/package-lock.json

```diff
@@ -3016,6 +2991,717 @@
             "glob": "^7.2.0",
             "mime": "^2.6.0",
             "yargs": "^16.2.0"
+          },
+          "dependencies": {
+            "@aws-cdk/cloud-assembly-schema": {
+              "version": "1.130.0",
+              "resolved": "https://registry.npmjs.org/@aws-cdk/cloud-assembly-schema/-/cloud-assembly-schema-1.130.0.tgz",
+              "integrity": "sha512-FewdApVM5N4BwNn23VeiIA6EgD6CZP1tV0wZAsxU+xXl4CwsfhyeAsmTVeHvf4+g+N7j5BBp+q2d1bkSesdZWw==",
+              "dev": true,
+              "requires": {
+                "jsonschema": "^1.4.0",
+                "semver": "^7.3.5"
```

Comment:
Not sure I love seeing this pull in an old version in the `package-lock.json`. Can you go ahead and:

```
$ git fetch
$ git checkout origin/develop -- infrastructure/package-lock.json
$ cd infrastructure
$ rm -rf node_modules
$ npm install
```

and just make sure that everything runs okay?

Reply:
Must've popped in during the merge. I just committed a fix for it.

Reply:
Fixed in 2119f057bac3b05c00540e396ec74e52db0e7cdd